src/ray/rpc/client_call.h

```diff
@@ -309,6 +331,9 @@ class ClientCallManager {
     }
   }
 
+  /// UUID of this generation of the cluster.
```

Comment:
what does this sentence mean?

Reply:
If the cluster restarts, it is a new generation of the same cluster... should it just be considered a "new cluster"?

Reply:
Yea, if it's GCS not backed by redis then after restart, it's a new cluster

Reply:
Removed "generation"